ansible/roles/data-products-deploy/defaults/main.yml

```diff
@@ -2,7 +2,7 @@ analytics_user: analytics
 analytics_group: analytics
 spark_output_temp_dir: /mount/data/analytics/tmp/
 
-bucket: "dev-data-store"
+bucket: "{{ env }}-data-store"
```

Comment:
Will this default the env to sunbirddev? I think the container we use in dev is dev-data-store and subsequently in other environments it is telemetry-data-store.

Reply:
Fixed - https://github.com/project-sunbird/sunbird-data-pipeline/pull/1283/commits/ebe712d6f8b579808f6ad3c2e70d53c79d54b200
Dev we will override private repo